canonical/vanilla-framework
# Add sections and blocks components

## Done

Added three new classes: `.p-block`, `.p-section` for managing spacing between different types of elements on the page.

Fixes [WD-3019](https://warthogs.atlassian.net/browse/WD-3019)

## QA

- Open [demo](https://vanilla-framework-4730.demos.haus/docs/patterns/section)
- Review updated examples:
  - [blocks](https://vanilla-framework-4730.demos.haus/docs/examples/patterns/section-and-block/block)
  - [sections](https://vanilla-framework-4730.demos.haus/docs/examples/patterns/section-and-block/section)
- Review updated documentation:
  - [sections and blocks](https://vanilla-framework-4730.demos.haus/docs/patterns/section)
- Check if brochure site docs and examples use new section/block components:
  - https://vanilla-framework-4730.demos.haus/docs/layouts/brochure
  - https://vanilla-framework-4730.demos.haus/docs/examples/layouts/brochure-site/index
 
### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical/vanilla-framework/wiki/Release-process#pre-release-tasks)



[WD-3019]: https://warthogs.atlassian.net/browse/WD-3019?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
